src/com/goide/quickfix/GoStringIndexIsByteQuickFix.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2013-2016 Sergey Ignatov, Alexander Zolotov, Florin Patan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.goide.quickfix;
+
+import com.goide.GoTypes;
+import com.goide.psi.GoStringLiteral;
+import com.intellij.codeInspection.LocalQuickFixOnPsiElement;
+import com.intellij.lang.ASTFactory;
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.impl.source.tree.FileElement;
+import com.intellij.psi.impl.source.tree.LeafElement;
+import com.intellij.util.IncorrectOperationException;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+
+import static com.intellij.psi.impl.GeneratedMarkerVisitor.markGenerated;
+import static com.intellij.psi.impl.source.DummyHolderFactory.createHolder;
+
+public class GoStringIndexIsByteQuickFix extends LocalQuickFixOnPsiElement {
```

Comment:
you can save some memory by reimplementing a fix with LocalQuickFixBase and using a descriptor for retrieving target element